### ISSUE TITLE
CLI: No feedback when referencing a non-existent policy file

### ISSUE THREAD

**Issue by nkrabben** [MediaArea/MediaConch#199]
When I use the CLI with a local policy, but mispell the path, MediaConch doesn't report that it failed to load the policy. Instead it falls back to default policies, and prints a blank line.

For example, here I don't complete the policy's filename.
`
>mediaconch -p policy.xm media_file.mov
pass! media_file.mov

`

This is what a successful test looks like
`
>mediaconch -p policy.xml media_file.mov
pass! media_file.mov
`

It would be helpful if MediaConch reported that no file exists when given a bad path for a policy, or that it is validating using the default policies.

**Comment by ablwr** [comment]
Ahhh this is a good catch. Can you also tell us your MediaConch version and OS you're working on?

**Comment by nkrabben** [comment]
MediaConch Command Line Interface 17.07
macOS 10.11.6

**Comment by JeromeMartinez** [comment]
Right, this was maybe not the smartest behavior, and we need to change that.
